app/src/main/java/org/sopt/at/SignUpActivity.kt

```diff
@@ -0,0 +1,334 @@
+                                contentDescription = "뒤로가기",
+                                tint = Color.White
+                            )
+                        }
+                    }
+                    SignUpView(
+                        modifier = Modifier.padding(innerPadding),
+                        onNext = onNext,
+                        signUp = signUp,
+                        step = step,
+                        isError = isError
+                    )
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun SignUpView(
+    modifier: Modifier,
+    onNext: (String) -> Unit,
+    signUp: (String, String) -> Unit,
+    step: String,
+    isError: Boolean
+) {
+    var id by remember { mutableStateOf("") }
+    var password by remember { mutableStateOf("") }
+    Column(
+        modifier = modifier
+            .fillMaxSize()
+            .background(color = Color.Black),
+        verticalArrangement = Arrangement.Top,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+
+        if (step == "아이디") {
+            IdInputView(
+                text = id,
+                onValueChange = { id = it },
+                onNext = { onNext(id) },
+                isError = isError
+            )
+        } else if (step == "비밀번호") {
+            PasswordInputView(
+                text = password,
+                onValueChange = { password = it },
+                onSignUp = { signUp(id, password) },
+                isError = isError
+            )
+        }
+    }
+}
+
+@Composable
+fun IdInputView(
+    text: String,
+    onValueChange: (String) -> Unit,
+    onNext: () -> Unit,
+    isError: Boolean
+) {
+    Column(
+        modifier = Modifier
+            .width(480.dp)
+            .padding(horizontal = 4.dp)
+    ) {
+        Text(
+            "아이디를 입력해주세요.",
+            fontSize = 24.sp,
+            fontWeight = FontWeight.Bold,
+            modifier = Modifier
+                .padding(bottom = 36.dp),
+            color = Color.White
+        )
+        OutlinedTextField(
+            value = text,
+            onValueChange = onValueChange,
+            placeholder = { Text("아이디", color = Color.LightGray) },
+            singleLine = true,
+            modifier = Modifier
+                .fillMaxWidth()
+                .clip(RoundedCornerShape(2.dp))
+                .background(color = Color.DarkGray)
+                .height(52.dp),
+            colors = OutlinedTextFieldDefaults.colors(
+                focusedBorderColor = Color.LightGray,
+                unfocusedBorderColor = Color.Transparent,
+                cursorColor = Color.White,
+            ),
+            shape = RoundedCornerShape(2.dp),
+            textStyle = TextStyle(color = Color.White)
+        )
+        Text(
+            "영문 소문자 또는 영문 소문자, 숫자 조합 6~12자리",
+            color = if (isError) {
+                Color.Red
+            } else {
+                Color.Gray
+            },
```

Comment:
요거 이렇게 쓰면 더 깔끔할 것 같습니다!
```suggestion
            val color = if (isError) Color.Red else Color.Gray
```